src/core/directives/google-map-marker.ts

```diff
@@ -38,7 +38,7 @@ let markerId = 0;
   selector: 'sebm-google-map-marker',
   inputs: [
     'latitude', 'longitude', 'title', 'label', 'draggable: markerDraggable', 'iconUrl',
-    'openInfoWindow', 'fitBounds', 'opacity', 'visible'
+    'openInfoWindow', 'fitBounds', 'opacity', 'visible', 'complexIcon'
```

Comment:
Why would you name it something other than 'icon'?  I think we should follow as close as possible to the actual google maps Marker type.